### DISCUSSION TITLE
Why isn't the time updated?

### DISCUSSION THREAD

**Discussion by doczi1123** [jomjol/AI-on-the-edge-device#768] [Q&A]
Hi!

Why isn't the time updated?

Steve

config.ini  file :
[System]
TimeZone = CET-1CEST,M3.5.0,M10.5.0/3
TimeServer = pool.ntp.org
;AutoAdjustSummertime = false

Log file :
12:08:33.343 -> I (6584) wifi:AP's beacon interval = 102400 us, DTIM period = 1
12:08:35.293 -> [0;32mI (8544) sntp: Time is not set yet. Connecting to WiFi and getting time over NTP.[0m
12:08:35.293 -> [0;32mI (8544) sntp: Initializing SNTP[0m
12:08:35.293 -> [0;32mI (8544) sntp: Waiting for system time to be set... (1/10)[0m
12:08:37.288 -> [0;32mI (10544) sntp: Waiting for system time to be set... (2/10)[0m
12:08:39.288 -> [0;32mI (12544) sntp: Waiting for system time to be set... (3/10)[0m
12:08:41.288 -> [0;32mI (14544) sntp: Waiting for system time to be set... (4/10)[0m
12:08:41.381 -> [0;32mI (14644) BLINK: Blinken - done[0m
12:08:43.281 -> [0;32mI (16544) sntp: Waiting for system time to be set... (5/10)[0m
12:08:45.279 -> [0;32mI (18544) sntp: Waiting for system time to be set... (6/10)[0m
12:08:47.276 -> [0;32mI (20544) sntp: Waiting for system time to be set... (7/10)[0m
12:08:49.271 -> [0;32mI (22544) sntp: Waiting for system time to be set... (8/10)[0m
12:08:51.273 -> [0;32mI (24544) sntp: Waiting for system time to be set... (9/10)[0m
12:08:51.458 -> [0;32mI (24734) BLINK: Blinken - done[0m
12:08:53.276 -> TimeZone set to CET-1CEST,M3.5.0,M10.5.0/3
12:08:53.276 -> Logfile opened: /sdcard/log/message/log_1970-01-01.txt
12:08:53.276 -> W (26554) wifi:<ba-add>idx:0 (ifx:0, 74:4d:28:45:e7:d1), tid:0, ssn:3, winSize:64
12:08:53.322 -> [0;32mI (26564) sntp: The current date/time in Berlin is: Thu Jan  1 03:00:40 1970[0m
12:08:53.322 -> [0;32mI (26564) sntp: The current date/time in Berlin is: 1970-01-01_03:00[0m
12:08:53.322 -> timeist 19700101-030040
12:08:53.322 -> Logfile opened: /sdcard/log/message/log_1970-01-01.txt
12:08:53.322 -> Logfile opened: /sdcard/log/message/log_1970-01-01.txt
12:08:53.368 -> Logfile opened: /sdcard/log/message/log_1970-01-01.txt
12:08:53.368 -> time 19700101-030040

**Comment by jomjol** [comment]
The ESP32 is not finding the connection to the time server. Reason hard to say. If you do not specify it in the config.ini, a default one will be use. Sometimes there is a problem with the DNS-server, especially with local hosts l like fritz box. You can specify a ip-adress directly in this case.
